pennylane_lightning/src/simulators/lightning_qubit/utils/tests/CMakeLists.txt

```diff
@@ -27,8 +27,7 @@ target_sources(lightning_qubit_utils_tests INTERFACE runner_lightning_qubit_util
 ################################################################################
 # Define targets
 ################################################################################
-set(TEST_SOURCES    Test_ConstantUtil.cpp
```

Comment:
Do we not test the compile-time/const functions anymore? Or is this just moved elsewhere?

Reply:
Moved to `pennylane_lightning/src/utils/tests/Test_ConstantUtil.cpp`.